content/en/users/aai/check-in/obtaining-tokens/oidc-agent/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: oidc-agent
+linkTitle: oidc-agent
+type: docs
+weight: 40
+aliases:
+  - /users/aai/check-in/obtaining-tokens/oidc-agent
+description: >
+  Usage guide of oidc-agent
+---
+
+The oidc-agent is a command-line tool for managing OpenID Connect tokens
+developed by Karlsruhe Institute of Technology (KIT).
```

Comment:
Some links in the intro would be nice (like to the GitHub repo and KIT home page)